proc.go

```diff
@@ -103,6 +103,17 @@ func (p Proc) CmdLine() ([]string, error) {
 	return strings.Split(string(data[:len(data)-1]), string(byte(0))), nil
 }
 
+// AbsCmdLine returns absolute path to the command line of a process.
+func (p Proc) AbsCmdLine() (string, error) {
+	ac, err := p.readlink("exe")
```

Comment:
Hmm, `exe` does not contain the process's command line, just the first part of it (the executable). Maybe it would make sense to rename this method to `Executable`?